src/components/Button.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+function Button({children, clickHandler}) {
```

Comment:
Ik had denk ik zelf niet gekozen om hier een apart component voor te maken. In principe heb je hiermee geen code ontdubbeld of gereduceerd